### PR TITLE
[onkyo] Add channel for changing the Audyssey EQ Mode

### PR DESCRIPTION
**Pull request by Albinoman887**
* Values are Off, Movie, and Music and and corspond to 0,1,2

Signed-off-by: Matt Filetto <matt.filetto@gmail.com>

### CODE REVIEW

**Comment by lolodomo** [comment]
Build failed. You need to apply spotless (your file channels.xml). Run "mvn spotless:apply"

File: bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -103,6 +103,18 @@
...
+		<label>Audyssey EQ Mode</label>
+		<description>Choose between Audyssey 2EQ/MultiEQ Equalizer curves</description>
+		<state>
+			<options>
+				<option value="0">Audyssey EQ Off</option>
+				<option value="1">Audyssey EQ Movie</option>
+				<option value="2">Audyssey EQ Music</option>
+			</options>
+		</state>
+    </channel-type>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml:117]: Please update the onkyo.properties file. Should be doable by just running "mvn i18n:generate-default-translations"

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java [hunk 2]
```diff
@@ -75,6 +75,7 @@ public class OnkyoBindingConstants {
     public static final String CHANNEL_LISTENMODE = "player#listenmode";
     public static final String CHANNEL_AUDIOINFO = "player#audioinfo";
     public static final String CHANNEL_PLAY_URI = "player#playuri";
+    public static final String CHANNEL_AUDYSSEYEQ = "player#audysseyeq";
```
Comment (lolodomo) [inline bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java:78]: Please also update README to present this new channel.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java (hunk 2 above)
Comment (Albinoman887) [reply bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java:78]: for some reason the emails from github where going to spam. i forgot about this. i can still make the requested changes if u want but should i still? or just make a PR for OH4 version

Comment (lolodomo) [reply bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java:78]: Yes, it is now expected that you test your code changes in OH 4.1 env.

**Comment by Albinoman887** [comment]
what do i need to do exactly?

**Comment by lsiepel** [comment]
> what do i need to do exactly?

This PR is 1200 commits behind, so to make a testable jar for the current builds, you would need to update your branches.
1. update your main branch (Albinoman887:main)  from upstream
2. update your feature branch (Albinoman887:main_onkyo-audysseyeq) from main
3. fix any merge conflict, (don't think there are any)
4. commit that merge.
5. address review comments + commit
6. create jar and test it

File: bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (Albinoman887) [reply bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml]: done

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java (hunk 2 above)
Comment (Albinoman887) [reply bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java:78]: k

**Comment by Albinoman887** [comment]
updated with all the changes

File: bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml [hunk 3]
```diff
@@ -103,6 +103,18 @@
...
 	</channel-type>
+	<channel-type id="audysseyeq" advanced="true">
+		<item-type>Number</item-type>
+		<label>Audyssey EQ Mode</label>
+		<description>Choose between Audyssey 2EQ/MultiEQ Equalizer curves</description>
+		<state>
+			<options>
+				<option value="0">Audyssey EQ Off</option>
+				<option value="1">Audyssey EQ Movie</option>
+				<option value="2">Audyssey EQ Music</option>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml:114]:
```suggestion
				<option value="0">Off</option>
				<option value="1">Movie</option>
				<option value="2">Music</option>
```

As the channel and label is allready Audyssey EQ, this duplicate info is aonly cluttering the ui. Please remove and also re-run the i18n plugin as you did before to also update the translation files.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks for picking this up again!
Overall looks good, left just one comment.

File: bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml (hunk 3 above)
Comment (Albinoman887) [reply bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml]: done

**Comment by Albinoman887** [comment]
updated

**Review by Albinoman887 — COMMENTED** [review]
good?

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Review by Albinoman887 — COMMENTED** [review]
looks good to me

**Pull request merged**